src/engine/Marabou.cpp

```diff
@@ -144,11 +144,32 @@ void Marabou::displayResults( unsigned long long microSecondsElapsed ) const
         for ( unsigned i = 0; i < _inputQuery.getNumInputVariables(); ++i )
             printf( "\tx%u = %lf\n", i, _inputQuery.getSolutionValue( _inputQuery.inputVariableByIndex( i ) ) );
 
-        printf( "\n" );
-        printf( "Output:\n" );
-        for ( unsigned i = 0; i < _inputQuery.getNumOutputVariables(); ++i )
-            printf( "\ty%u = %lf\n", i, _inputQuery.getSolutionValue( _inputQuery.outputVariableByIndex( i ) ) );
-        printf( "\n" );
+        if ( _inputQuery._networkLevelReasoner )
+        {
+            double input[_inputQuery.getNumInputVariables()];
```

Comment:
This is not a legal static array declaration, since the value is not known at compilation time. One needs to allocate memory on the heap for this.